examples/enable-gcp-services-simple/README.md

```diff
@@ -0,0 +1,33 @@
+# enable-gcp-services: Simple Example
+
+### Overview
+
+In this example, we will see how to generate [project services](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service) for `compute.googleapis.com` and `redis.googleapis.com`.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/enable-gcp-services-simple
+```
+
+### Function invocation
+
+Render the configuration, `enable-gcp-services` has been defined declaratively in the Kptfile and configured via `services.yaml`.
+
+```shell
+$ kpt fn render enable-gcp-services-simple
```

Comment:
It may be helpful to let users run it with `--results-dir results` and then let the users check the structured results.
WDYT?

Reply:
I dropped this per https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/568#discussion_r705514638
We surface results via stdout and I don't see much value in this simple example to examine structured results.